man/bundle-install.ronn

```diff
@@ -47,7 +47,12 @@ To apply any of `--deployment`, `--path`, `--binstubs`, or `--without` every
 time `bundle install` is run, use `bundle config` (see bundle-config(1)).
 
 * `--binstubs[=<directory>]`:
-  Creates a directory (defaults to `~/bin`) and place any executables from the
+  Binstubs are scripts that wrap around executables. Bundler creates a small Ruby
+  file (a binstub) that loads Bundler, runs the command, and puts it in `bin/`.
+  This lets you link the binstub inside of an application to the exact gem
+  version the application needs.
+  
```

Comment:
There is a whitespace here as well that needs to be removed.

Reply:
@colby-swandale I'm not seeing the extra whitespace—mind pointing it out?

Reply:
There is a space character on line 54 that shouldn't be there (which is why the build is failing in CI)